sorting/external_sort.cpp

```diff
@@ -0,0 +1,194 @@
+/**
+ * @file
+ * @brief [External Sorting] https://en.wikipedia.org/wiki/External_sorting
+ * @details
+ * C++ program to sort a file data using external sorting
+ * External sorting is required when the data being sorted do not fit into the
+ * memory (RAM)
+ * @author [Hoai Le](https://github.com/bls-lehoai)
+ */
+
+#include <algorithm>  // for in-memory sorting
+#include <cassert>    // for testing
+#include <fstream>    // for read/write data file
+#include <iostream>   // for I/O operations
+#include <random>     // for generate a random number
+#include <string>     // for string operations (read file line by line)
+
+using namespace std;
```

Comment:
```suggestion

```
avoid using namespace std;

Reply:
@realstealthninja 
sorry, why we can't use namespace std?